deployment/stacks/parent.ts

```diff
@@ -0,0 +1,139 @@
+import { Construct } from "constructs";
+import {
+  Aws,
+  aws_ec2 as ec2,
+  aws_iam as iam,
+  aws_s3 as s3,
+  CfnOutput,
+  Stack,
+  StackProps,
+} from "aws-cdk-lib";
+import { DeploymentConfigProperties } from "../deployment-config";
+import { createDataComponents } from "./data";
+import { createUiAuthComponents } from "./ui-auth";
+import { createUiComponents } from "./ui";
+import { createApiComponents } from "./api";
+import { deployFrontend } from "./deployFrontend";
+import { createCustomResourceRole } from "./customResourceRole";
+import { isLocalStack } from "../local/util";
+import { createTopicsComponents } from "./topics";
+import { createUploadsComponents } from "./uploads";
+import { getSubnets } from "../utils/vpc";
+
+export class ParentStack extends Stack {
+  constructor(
+    scope: Construct,
+    id: string,
+    props: StackProps & DeploymentConfigProperties
+  ) {
+    const {
+      isDev,
+      secureCloudfrontDomainName,
+      vpcName,
+      kafkaAuthorizedSubnetIds,
+    } = props;
+
+    super(scope, id, {
+      ...props,
+      terminationProtection: !isDev,
+    });
+
+    const iamPermissionsBoundaryArn = `arn:aws:iam::${Aws.ACCOUNT_ID}:policy/cms-cloud-admin/developer-boundary-policy`;
+    const iamPath = "/delegatedadmin/developer/";
+
+    const commonProps = {
+      scope: this,
+      ...props,
+      iamPermissionsBoundary: iam.ManagedPolicy.fromManagedPolicyArn(
+        this,
+        "iamPermissionsBoundary",
+        iamPermissionsBoundaryArn
+      ),
+      iamPath,
+    };
+
+    const vpc = ec2.Vpc.fromLookup(this, "Vpc", { vpcName });
+    const kafkaAuthorizedSubnets = getSubnets(
+      this,
+      kafkaAuthorizedSubnetIds ?? ""
```

Comment:
Since we validate that this will be there in deployment config I think we don't need to do the ?? "" part.

Reply:
I think we if switch to this in the app.ts this problem goes away:
```
  if (stage == "bootstrap") {
    new Stack(app, `${config.project}-${stage}`, {});
```
That's how it is in SEDS and HCBS now.

Reply:
Fixed.